src/js/options.js

```diff
@@ -168,6 +168,22 @@ function loadOptions() {
       });
   }
 
+  // only show the x-client-data header setting if in Chrome & Chromium browsers
+  // TODO: more accurate way to determine this is a Chrome or Chromium browser
```

Comment:
`X-Client-Data` is not in Chromium, only Chrome, I think.

We could try doing feature detection (always better than guessing based on UA). Something like, on Privacy Badger startup, make a dummy request to a Google domain. This request should get cancelled, but before it does, we'll see the headers and set our internal `xClientDataHeaderDetected` flag.